research-software-directory/RSD-as-a-service
# 169 organisation

# Organisation page and research units

Fixes #169 

Changes proposed in this pull request:

*  When clicking on the organisation from the organisation overview the user will load organisation page with software,projects and research units (if any) of the organisation. On organisation page user can search for software or projects within selected organisation.
*  Project overview page with basic search
*  Creating research units. Maintainers of organisation can add new research units and change organisation properties like name, ROR,website en logo van research unit. De research unit is placed as "subsidiary" of "main" organisation. **These sub-organisations (research units) are not shown as organisations in the organisation overview page, nor they can be found using search option on the organisation overview page.**

How to test:
* `docker-compose down --volumes`: remove old instance
* `docker-compose build`: build all images
* `docker-compose up`: start new instance
* perform data-migration
* go to organisation page and select top organisation, you should see similar screen as below and you can navigate through software and projects of the organisation

![image](https://user-images.githubusercontent.com/9204081/161078286-d4bac2ad-44ce-4bcd-a21d-7978238e2f7a.png)

* visit projects page, you should see similair output as in image below. You should be able to navigate and search on project title and subtitle values

![image](https://user-images.githubusercontent.com/9204081/161086343-e8f214d5-523e-4a3f-afc4-458785598564.png)

PR Checklist:
